cogs/get_token_authorisation.py

```diff
@@ -60,17 +58,21 @@ async def get_token_authorisation(self, ctx: TeXBotApplicationContext) -> None:
         async with http_session, http_session.get(REQUEST_URL) as http_response:
             response_html: str = await http_response.text()
 
-        parsed_html: bs4.Tag | bs4.NavigableString | None = BeautifulSoup(
+        page_title: bs4.Tag | bs4.NavigableString | None = BeautifulSoup(
             response_html,
             "html.parser",
-        ).find("ul", {"id": "ulOrgs"})
+        ).find("title")
+
+        if not page_title:
+            await self.command_send_error(ctx, message="Something has gone very wrong.")
```

Comment:
Improved error message should be more specific, possibly about the page being empty.